models/articles.model.js

```diff
@@ -2,11 +2,30 @@ const { normalizeQueryConfig } = require("pg/lib/utils");
 const db = require("../db/connection");
 const fs = require("fs/promises");
 
-const fetchArticles = () => {
+const fetchArticles = (query = { sort_by: "created_at", order: "DESC" }) => {
+  let queryStr =
+    "SELECT articles.article_id, articles.author, articles.title, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count  FROM articles  LEFT OUTER JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id";
```

Comment:
You can condense the selection of all columns in articles by using **articles.*** 💡 This will make your query more readable.